--- a/internal/server/server.go
+++ b/internal/server/server.go
@@ -111,7 +111,12 @@ func (s *Server) SetVersion(version string) {
 }

 // GetRTMService returns the server's RTM service.
-func (s *Server) GetRTMService() *rtm.Service {
+func (s *Server) GetRTMService() httputils.RTMServiceInterface {
     return s.rtmService
 }
+
+// GetTokenManager returns the token manager.
+func (s *Server) GetTokenManager() httputils.TokenManagerInterface {
+    return s.tokenManager
+}
